src/js/navButtons.js

```diff
@@ -96,7 +105,8 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             nextButtonId = fluid.allocateSimpleId(nextButton),
             showSelector = that.options.styles.show,
             isFirstPanel = currentPanelNum === that.options.panelStartNum,
```

Comment:
Not to grind away at such a tiny point, but how about the following - 
gpii.firstDiscovery.navButtons.indexToDisposition = function (currentPanelNum, panelTotalNum) {
    return currentPanelNum === 0 ? 0 : (currentPanelNum === panelTotalNum ? 2 : 1);
    }

    var disposition = gpii.firstDiscovery.navButtons.indexToDisposition(currentPanelNum, panelTotalNum);
    nextLabel = strings[ ["first", "next", "last"][disposition] ];
    nextTooltipContent = strings[ ["firstTooltip", "nextTooltip", "lastTooltip"][disposition] ]